project/.classpath

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
also lets add .classpath and .project to gitignore as these are also related to eclipse jdk tooling 
(same instructions: https://www.atlassian.com/git/tutorials/saving-changes/gitignore just try adding:
.classpath
.project 
to the gitignore file